nequip/data/AtomicData.py

```diff
@@ -249,6 +250,44 @@ def from_ase(cls, atoms, r_max, **kwargs):
             **add_fields,
         )
 
+    def to_ase(self) -> Union[List[ase.Atoms], ase.Atoms]:
```

Comment:
since you `from ase import Atoms`, the name for `ase.Atoms` in this file is `Atoms`... I guess. Probably cleanest (in my opinion) to change all instances of `Atoms` in this file to `ase.Atoms` and replace `from ase import Atoms` with `import ase`.

While opinions differ on this, I tend to be of the opinion that `from whatever import things` should be avoided if it doesn't spare a lot of typing — makes the code much easier to read when you know that you're talking about ASE's Atoms, or `torch`'s Tensor, and not something else